simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/component/chargingstation/ChargeProtocolVariableAccessorTest.java

```diff
@@ -0,0 +1,72 @@
+package com.evbox.everon.ocpp.simulator.station.component.chargingstation;
+
+import com.evbox.everon.ocpp.common.CiString;
+import com.evbox.everon.ocpp.simulator.station.StationHardwareData;
+import com.evbox.everon.ocpp.v20.message.centralserver.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.stream.Stream;
+
+import static com.evbox.everon.ocpp.simulator.assertion.CiStringAssert.assertCiString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+@ExtendWith(MockitoExtension.class)
+class ChargeProtocolVariableAccessorTest {
+
+    @InjectMocks
+    ChargeProtocolVariableAccessor variableAccessor;
+
+    static Stream<Arguments> setVariableDatumProvider() {
+        return Stream.of(
+                arguments(ChargingStationComponent.NAME, ChargeProtocolVariableAccessor.NAME, SetVariableDatum.AttributeType.ACTUAL, "some_value", SetVariableResult.AttributeStatus.REJECTED)
+        );
+    }
+
+    static Stream<Arguments> getVariableDatumProvider() {
+        return Stream.of(
+                arguments(ChargingStationComponent.NAME, ChargeProtocolVariableAccessor.NAME, GetVariableDatum.AttributeType.ACTUAL, GetVariableResult.AttributeStatus.ACCEPTED, StationHardwareData.PROTOCOL_VERSION),
```

Comment:
do we have line length enforcement in our coding standard?

Reply:
No, we don't at the moment. But I agree there are too many arguments.
I'll combine some of them to one object, but will do it in following PR to reduce merge conflicts

Reply:
I think intellij has 120 characters by default

Reply:
addressed it here: https://github.com/evbox/station-simulator/pull/11/files